next/components/forms/simple-components/Accordion.tsx

```diff
@@ -1,13 +1,22 @@
 import cx from 'classnames'
 import React, { useState } from 'react'
+import ReactMarkdown from 'react-markdown'
+import rehypeRaw from 'rehype-raw'
+import remarkDirective from 'remark-directive'
+import remarkDirectiveRehype from 'remark-directive-rehype'
+import remarkGfm from 'remark-gfm'
 
 import ExpandMoreIcon from '../icon-components/ExpandMoreIcon'
 import PersonIcon from '../icon-components/PersonIcon'
+import TooltipComponent from './Tooltip'
 
+const Tooltip = ({ children }: never) => {
+  return <TooltipComponent tooltip={children} />
+}
 type AccordionBase = {
   size: 'sm' | 'md' | 'lg'
   title: string
-  children: React.ReactNode
+  children: string
```

Comment:
let's rename this prop, perhaps to `markdownContent`

within react world, `children` prop has it's own special place and usually implies accepting react nodes